runtime/vm/NativeHelpers.cpp

```diff
@@ -149,6 +149,17 @@ convertCStringToByteArray(J9VMThread *currentThread, const char *cString)
 	return result;
 }
 
+UDATA *
+convertToNativeArgArray(J9VMThread *currentThread, j9object_t argArray, UDATA *javaArgs)
+{
+	UDATA argCount = (UDATA)J9INDEXABLEOBJECT_SIZE(currentThread, argArray);
+	UDATA *result = javaArgs;
+	if (NULL != result) {
+		VM_ArrayCopyHelpers::memcpyFromArray(currentThread, argArray, (UDATA)3, 0, argCount, (void*)result);
```

Comment:
Some description of the magic "3" would be nice.

Reply:
Comment was added for `3` (3 means 8 bytes for each element of the array to be copied).